src/rust/en.webtoonxyz/res/source.json

```
@@ -0,0 +1,31 @@
+{
+	"info": {
+		"id": "en.webtoonxyz",
+		"lang": "en",
+		"name": "WebtoonXYZ",
+		"version": 1,
+		"url": "https://webtoon.xyz",
+		"nsfw": 1
```

honestly it looks like primarily nsfw stuff to me
```suggestion
		"nsfw": 2
```